### PR TITLE
mariadb: support custom my.cnf config in values

### LINKED ISSUE #71: MariaDB with custom configuration extensions
I have to supply MariaDB with custom configuration extensions/overrides. For example:

``` ini
[mysqld]
max_allowed_packet = 64M
sql_mode=STRICT_ALL_TABLES
ft_stopword_file=/etc/mysql/stopwords.txt
ft_min_word_len=3
ft_boolean_syntax=' |-><()~*:""&^'
innodb_buffer_pool_size=2G
```

How do I do this with the MariaDB Chart?

### PR DESCRIPTION
**Pull request by prydonius**
Adds the config property to `Values.yaml` that gets embedded in a
configMap and mounted as a custom `my.cnf` file.

Closes #71

### CODE REVIEW

File: stable/mariadb/README.md [hunk 1]
```diff
@@ -85,6 +86,28 @@ $ helm install --name my-release -f values.yaml mariadb-x.x.x.tgz
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+### Custom my.cnf configuration
+
+The Bitnami MariaDB image allows you to provide a custom `my.cnf` file for configuring MariaDB.
+This Chart uses the `config` value to mount a custom `my.cnf` using a ConfigMap.
```
Comment (chrislovecnm) [inline stable/mariadb/README.md]: Would it be good to link to the configmap docs here?

**Review by chrislovecnm — COMMENTED** [review]
Just the one comment about documentation. If you want to file my comment as an issue against the chart, I am good with that was well. Looks great, I like the pattern that is being used here, all kinds of stuff that we can do with this.

**Comment by prydonius** [comment]
Thanks @chrislovecnm! I've updated the README to point to the ConfigMap documentation 👍

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

**Comment by prydonius** [comment]
@viglesiasce @chrislovecnm  PTAL

**Comment by viglesiasce** [comment]
@k8s-bot test this

**Comment by viglesiasce** [comment]
@prydonius LGTM can you sign the CNCF CLA?

**Comment by prydonius** [comment]
I signed it!

**Pull request merged**